scipy/stats/_distn_infrastructure.py

```diff
@@ -2602,7 +2602,7 @@ def _drv2_ppfsingle(self, q, *args):  # Use basic bisection algorithm
             return c
 
 
-def entropy(pk, qk=None, base=None):
+def entropy(pk, qk=None, base=None, axis=0):
     """Calculate the entropy of a distribution for given probability values.
 
     If only probabilities `pk` are given, the entropy is calculated as
```

Comment:
This section of the docs explicity says `axis=0`. Perhaps it should be updated to `axis=axis`?

Reply:
done